components/chassis.py

```diff
@@ -381,6 +382,16 @@ def drive_local(self, vx: float, vy: float, omega: float) -> None:
         """Robot oriented drive commands"""
         self.chassis_speeds = ChassisSpeeds(vx, vy, omega)
 
+    def align_on_y(self, offset: float, precision: float) -> None:
+        if abs(offset) > precision:
+            self.chassis_speeds.vy = self.ALIGN_Y_AXIS_SPEED * -math.copysign(1, offset)
```

Comment:
The copy sign function allows you to apply the sign to the value within the function itself 
```suggestion
            self.chassis_speeds.vy =  -math.copysign(self.ALIGN_Y_AXIS_SPEED , offset)
```